src/api/app/controllers/webui/package_controller.rb

```diff
@@ -579,8 +579,12 @@ def rpmlint_result
     if @repo_list.empty?
       render partial: 'no_repositories', locals: { project: @project }
     else
-      render partial: 'rpmlint_result', locals: { index: params[:index], project: @project, package: @package,
-                                                  repository_list: @repo_list, repo_arch_hash: @repo_arch_hash }
+      # TODO: this is part of the temporary changes done for 'request_show_redesign'.
```

Comment:
Do we need to refactor this later? If so, create a card, please.

Reply:
Talked with @eduardoj offline about this. We're gonna sweep all the TODO's regarding the 'requet_show_redesign' at once, later, in one single card. No need to create a card to track this down.